### PR TITLE
Setup for react-redux

### PR DESCRIPTION
**Pull request by ryutaudo**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,8 +27,11 @@
     "morgan": "^1.9.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-redux": "^5.0.6",
     "react-scripts": "^1.0.17",
-    "redis": "^2.8.0"
+    "redis": "^2.8.0",
```
Comment (duyoji) [inline package.json]: Do you need a redis module in front part.

**Comment by ryutaudo** [comment]
I will create directories src/__test__

**Comment by duyoji** [comment]
@ryutaudo 

> I will create directories src/test

https://facebook.github.io/jest/

image

src/test => src/__tests__

**Comment by duyoji** [comment]
@ryutaudo 

You can add `__tests__` in this PR. because this PR is for just setting up environment.

**Comment by ryutaudo** [comment]
@duyoji 
Deleted files in tests.
No error is thrown on this PR in Heroku now.
Please check the code.

I will add test files when I need

**Comment by duyoji** [comment]
@ryutaudo 

`__tests__` directory is now removed from git history. Maybe Because of no file in __tests__ directory.

https://qiita.com/ndxbn/items/f124d2b183b60cb074e2

**Comment by ryutaudo** [comment]
@duyoji 
Passed tests for both local and Heroku. 
Now this branch is ready for review

File: package.json [hunk 2]
```diff
@@ -22,13 +22,19 @@
...
+    "enzyme-adapter-react-16": "^1.1.0",
     "express": "^4.16.2",
     "jest": "^21.2.1",
     "morgan": "^1.9.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-redux": "^5.0.6",
     "react-scripts": "^1.0.17",
-    "redis": "^2.8.0"
+    "react-test-renderer": "^16.2.0",
```
Comment (duyoji) [inline package.json]: Do you use this module?

Comment (ryutaudo) [reply package.json]: @duyoji no error happened on my end. So I will fix this.

**Comment by duyoji** [comment]
👍

**Pull request merged**